content/influxdb/cloud-dedicated/get-started/write.md

```diff
@@ -289,6 +212,254 @@ home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
+{{% /tab-content %}}
+{{% tab-content %}}
+<!----------------------------- BEGIN GO CONTENT ------------------------------>
+
+To write data to InfluxDB Cloud Dedicated using Go, use the
+[influxdb-client-go module](https://github.com/influxdata/influxdb-client-go).
+
+1.  Create a new go module in your project directory.
+
+    1. Create a new module directory and navigate into it.
+    2. Initialize a new Go module in the current working directory.
+    3. Create a `main.go` file.
+
+    ```sh
+    mkdir influxdb_go_client && cd $_
+    go mod init influxdb_go_client
+    touch main.go
+    ```
+
+2.  Install the [influxdb-client-go module](https://github.com/influxdata/influxdb-client-go).
+
+    ```sh
+    go get github.com/influxdata/influxdb-client-go/v2
+    ```
+
+3.  Inside of `main.go` instantiate an InfluxDB write client to write the
+    [line protocol above](#home-sensor-data-line-protocol) to InfluxDB.
+
+    1.  Import the following packages
+
+        - `context`
+        - `fmt`
+        - `log`
+        - `os`
+        - `time`
+        - `github.com/influxdata/influxdb-client-go/v2` aliased as `influxdb2`
+    
+    2.  Create a `dbWrite` function.
+    3.  In the `dbWrite` function, use `influxdb2.NewClientWithOptions` to
+        create a `writeClient` that accepts write options.
+        The write client requires the following credentials:
+        
+        - **url**: InfluxDB Cloud Dedicated cluster URL
+        - **token**: [Database token](/influxdb/cloud-dedicated/admin/tokens/)
+          with write access to the **get-started** database.
+          _For security reasons, we recommend setting this as an environment
+          variable rather than including the raw token string._
+
+        Because the timestamps in the line protocol are in second
+        precision, **use `SetPrecision(time.Second)` to define the write precision option**.
+    
+    4.  Use `writeClient.WriteAPIBlocking` to define a `writeAPI`.
+        The write API requires the following credentials:
+
+        - **org**: _Leave as an empty string_
+        - **bucket**: InfluxDB database name.
+    
+    5.  Define an array of line protocol strings where each element is a single
+        line of line protocol.
+    
+    6.  Iterate through the array of line protocol and use `writeAPI.WriteRecord`
+        to write each line of line protocol to InfluxDB.
+    
+    7.  Define a `main` function that executes the `dbWrite` function.
+
+{{% influxdb/custom-timestamps %}}
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+)
+
+func dbWrite(ctx context.Context) error {
+	// Create write client
+	url := "https://us-east-1-1.aws.cloud2.influxdata.com"
+	token := os.Getenv("INFLUX_TOKEN")
+	writeClient := influxdb2.NewClientWithOptions(url, token, influxdb2.DefaultOptions().SetPrecision(time.Second))
+
+	// Define write API
+	org := ""
```

Comment:
In my experience, `org` can't be empty.

Reply:
K, I'll note that this should be an arbitrary string.